discord/state.py

```diff
@@ -772,21 +772,16 @@ def parse_message_delete_bulk(self, data) -> None:
                 self._messages.remove(msg)  # type: ignore
 
     def parse_message_update(self, data) -> None:
-        raw = RawMessageUpdateEvent(data)
-        message = self._get_message(raw.message_id)
-        if message is not None:
-            older_message = copy.copy(message)
-            raw.cached_message = older_message
-            self.dispatch("raw_message_edit", raw)
-            message._update(data)
-            # Coerce the `after` parameter to take the new updated Member
-            # ref: #5999
-            older_message.author = message.author
-            self.dispatch("message_edit", older_message, message)
+        old_message = self._get_message(raw.message_id)
```

Comment:
shouldn't `self._messages` replace the old message with the new message?

Reply:
Ohy yeah right